docs/p0-guide-templates/Page_2-Setup.md

```diff
@@ -0,0 +1,48 @@
+
+# DataHub [Source Name] UI Ingestion Guide: Setup & Prerequesites
+
+<!-- In order to configure ingestion from [Source], you'll first have to ensure some basic prequesites are met -->
+
+## [Source Name] Prerequesites
+<!-- This section should provide plain-language instructions on any permissions/configuration need to occur in the Source:
+
+* What special permissions are required, if any? 
+* Screenshots of [Source] UI
```

Comment:
Yes this is very important!

Reply:
@gabe-lyons Did mention that customers are confused about what to do on the external platforms...